File: evidence/generated-systems/golang/hr-on-golang/hr-on-golang/internal/model/Organization.go
package model

import (
    "gorm.io/gorm"
)

//==============================================================
// Organization Declaration
//==============================================================
type Organization struct {
    gorm.Model
     Name                                    string
    LegalName                                    string
    RegistrationCountry                                    string
    Website                                    string
     Departments           []Department `gorm:"foreignKey:DepartmentsFromOrganizationId"`
     Locations           []Location `gorm:"foreignKey:LocationsFromOrganizationId"`
     JobFamilies           []JobFamily `gorm:"foreignKey:JobFamiliesFromOrganizationId"`
     BenefitPlans           []BenefitPlan `gorm:"foreignKey:BenefitPlansFromOrganizationId"`
     CostCenters           []CostCenter `gorm:"foreignKey:CostCentersFromOrganizationId"`
     PayrollCalendars           []PayrollCalendar `gorm:"foreignKey:PayrollCalendarsFromOrganizationId"`

// parent associations as their child

}

File: evidence/generated-systems/rubyonrails/hr-on-rails/hronrails/app/models/organization.rb
class Organization < ApplicationRecord


  has_many :Departments, class_name: 'Department'
  has_many :Locations, class_name: 'Location'
  has_many :JobFamilies, class_name: 'JobFamily'
  has_many :BenefitPlans, class_name: 'BenefitPlan'
  has_many :CostCenters, class_name: 'CostCenter'
  has_many :PayrollCalendars, class_name: 'PayrollCalendar'

end
